### ISSUE TITLE
Unable to start with the getting started guide on Mac OS X - boot errors

### ISSUE THREAD

**Issue by mrchrisadams** [hashobject/perun#244]
Hi there,

I'm on a mac with OS X 10.15.3, and I'm using boot for the first time.

I've tried to follow the steps outlined in the [getting started][1] guide and I think

a) the instructions are a bit of date, and 
b) I need some help, but I'm not sure the issue is this repo or a deeper one with boot

I'll start with the first one.

### Boot fails, looking for `src`

The first issue I cam across was an error about the src folder being missing. This was easy enough to fix, and would be a trivial PR, which I'm happy to make once I understand how this works.

### Boot fails with a `Must hint overloaded method: toArray` error

The second, I'm less sure of. I've [linked to the full traceback in this gist on github][2], but I think the key issue is this line here:

```
java.lang.IllegalArgumentException: Must hint overloaded method: toArray
        clojure.lang.ExceptionInfo: Must hint overloaded method: toArray
```

I had an earlier version of java before, which I've updated to 

*Investigating this further*

I've looked into this to see if this is an issue about boot rather than perun. I see a few problems reported on MacOS 10.14, and I've updated from Oracle:

```
 java --version                                                                                                                                           
java 14 2020-03-17
Java(TM) SE Runtime Environment (build 14+36-1461)
Java HotSpot(TM) 64-Bit Server VM (build 14+36-1461, mixed mode, sharing)
```

And I've also tried with the openjdk version:

```
java -version 
openjdk version "13.0.2" 2020-01-14
OpenJDK Runtime Environment (build 13.0.2+8)
OpenJDK 64-Bit Server VM (build 13.0.2+8, mixed mode, sharing

```

In both cases I get the same error, and I seem the same issue when I try with boot 2.5.5, 2.8.2  and 2.8.3.

I don't know enough about boot or perun to know where to troubleshoot next - any pointers?

[1]: https://github.com/hashobject/perun/wiki/Getting-Started
[2]: https://gist.github.com/mrchrisadams/d5ad9412c8a2e77f974299a90d0861d6

**Comment by allentiak** [comment]
Hi, Chris (@mrchrisadams),

Thanks for your detailed report!

Regarding (1): Please, feel free to open a PR with the specific parts where the documentation is incorrect. AFAIK, not many perun users use MacOS. Please note that the PR doesn't need to be complete - if (1) has something to do with (2), chances are both bugs will end up being solved simultaneously.

Regarding (2): I checked your logs, but wouldn't know where to begin... Exactly at which point are you stuck? Please see (1).

Best,
Leandro

**Comment by mrchrisadams** [comment]
Hi Leandro!

I've opened PR #245 now, which I think should help. If you're happy with merging that in, I'll update the wiki to point to the newer guide as well, has having the information duplicated, or at least missing any reference to newer content could result in someone else getting confused like me 👍

**Comment by allentiak** [comment]
@mrchrisadams Now that #245 has been merged... Is this bug still present?

**Comment by mrchrisadams** [comment]
Hi @allentiak thanks for checking on this. I think this is resolved now, by using the 0.4.3 release.

Have a good weekend 👍

**Comment by allentiak** [comment]
OK.

BTW, @mrchrisadams: We are looking for volunteers. I am planning a new "unstable" release in the upcoming weeks to include all the things on master since our last release.
 If you are interested to contribute, your PRs are more than welcome!
